client/wildcard/src/components/Button/story/Button.story.tsx

```diff
@@ -137,3 +127,82 @@ AllButtons.parameters = {
+
+                <ButtonGroup>
+                    <Button variant="secondary" outline={selectedButton !== 1} onClick={() => setSelectedButton(1)}>
+                        One
+                    </Button>
+                    <Button variant="secondary" outline={selectedButton !== 2} onClick={() => setSelectedButton(2)}>
+                        Two
+                    </Button>
+                    <Button variant="secondary" outline={selectedButton !== 3} onClick={() => setSelectedButton(3)}>
+                        Three
+                    </Button>
+                </ButtonGroup>
+            </div>
+
+            <div className="mb-4">
+                <H2>With Tooltips</H2>
+
+                <ButtonGroup>
+                    <Tooltip content="Option One">
+                        <Button variant="secondary" outline={selectedButton !== 1} onClick={() => setSelectedButton(1)}>
+                            One
+                        </Button>
+                    </Tooltip>
+                    <Tooltip content="Option Two">
+                        <Button variant="secondary" outline={selectedButton !== 2} onClick={() => setSelectedButton(2)}>
+                            Two
+                        </Button>
+                    </Tooltip>
+                    <Tooltip content="Option Three">
+                        <Button variant="secondary" outline={selectedButton !== 3} onClick={() => setSelectedButton(3)}>
+                            Three
+                        </Button>
+                    </Tooltip>
+                </ButtonGroup>
+            </div>
+
+            <div className="mb-4">
+                <H2>With Tooltips (Disabled Buttons)</H2>
+
+                <ButtonGroup>
+                    <Tooltip content="Option One">
+                        <Button variant="secondary" outline={selectedButton !== 1} onClick={() => setSelectedButton(1)}>
+                            One
+                        </Button>
+                    </Tooltip>
+                    <Tooltip content="Option Two">
+                        <Button
+                            disabled={true}
+                            variant="secondary"
+                            outline={selectedButton !== 2}
+                            onClick={() => setSelectedButton(2)}
+                        >
+                            Two
+                        </Button>
+                    </Tooltip>
+                    <Tooltip content="Option Three">
+                        <Button
+                            disabled={true}
+                            variant="secondary"
+                            outline={selectedButton !== 3}
+                            onClick={() => setSelectedButton(3)}
+                        >
+                            Three
+                        </Button>
+                    </Tooltip>
+                </ButtonGroup>
+            </div>
+        </>
+    )
+}
+Group.storyName = 'Button Group'
```

Comment:
```suggestion
Group.storyName = 'Button Group'
Group.parameters = {
    chromatic: {
        enableDarkMode: true,
        disableSnapshot: false,
    },
}
```

So we get visual regression tests in Chromatic on this story too!

Reply:
Good catch! Thanks!